applications/nrf_desktop/doc/dvfs.rst

```diff
@@ -0,0 +1,104 @@
+The module listens for Even Manager events that are sent and turns state on/off according to the event.
+It support the following DVFS states:
+
+* ``DVFS_STATE_INITIALIZING`` - State indicating that core initialization is in process.
+* ``DVFS_STATE_LLPM_CONNECTED`` - State indicating that Bluetooth Low Latency Packet Mode is active.
+* ``DVFS_STATE_USB_CONNECTED`` - State indicating that USB is connected.
+* ``DVFS_STATE_CONFIG_CHANNEL`` - State indicating that Config Channel is active.
+* ``DVFS_STATE_SMP_TRANSFER`` - State indicating that SMP transfer is active.
+
+Each DVFS state can be configured using the following Kconfig options:
+
+* ``CONFIG_DESKTOP_DVFS_STATE_<STATE>_ENABLE`` - Enable DVFS state <STATE>.
+* ``CONFIG_DESKTOP_DVFS_STATE_<STATE>_ACTIVE_FREQ`` - Choice of active frequency of state <STATE>.
+    The possible values are:  ``DESKTOP_DVFS_STATE_<STATE>_ACTIVE_FREQ_HIGH`` and ``DESKTOP_DVFS_STATE_<STATE>_ACTIVE_FREQ_MEDLOW``.
+* ``CONFIG_DESKTOP_DVFS_STATE_<STATE>_TIMEOUT_MS`` - Timeout in milliseconds specifying time from receiving on event to moment the state <STATE> stops requesting its active frequency.
+    When the timeout is reached, the state will go back to a nonactive state potentially enabling the system to go to lower frequency.
+    If timeout is set to 0 the <STATE> will not cease requesting its active frequency until receiving off event.
+
+Each change request is retried if it fails due to known reason.
+The module will retry the request up to the number of times specified by :option:`CONFIG_DESKTOP_DVFS_RETRY_COUNT`.
+Error counter is only zeroed when preforming correct DVFS change request.
+Failing frequency change requests can be caused by another frequency change request being in progress on sysctrl - error ``-EBUSY`` or by the nrfs DVFS service being not yet initialized - error ``-EAGAIN``.
+If the frequency change request is in progress, the module will retry the request after a timeout specified by :option:`CONFIG_DESKTOP_DVFS_RETRY_BUSY_TIMEOUT_MS`.
+If the nrfs service DVFS is not yet initialized, the nRF Desktop DVFS module will retry the request after a timeout specified by :option:`CONFIG_DESKTOP_DVFS_RETRY_INIT_TIMEOUT_MS`.
+If the DVFS frequency request returns an error different than ``-EBUSY`` or ``-EAGAIN``, the module will report ``MODULE_STATE_ERROR`` and set ``module_state`` to ``STATE_ERROR``.
+nRF Desktop DVFS module will also report MODULE_STATE_ERROR and set ``module_state`` to ``STATE_ERROR`` if the number of retries exceeds the value specified by :option:`CONFIG_DESKTOP_DVFS_RETRY_COUNT`.
+In ``module_state`` in ``STATE_ERROR`` the module will not change the frequency and voltage.
+
+* :ref:`CONFIG_DESKTOP_DVFS_RETRY_BUSY_TIMEOUT_MS <config_desktop_app_options>` - Timeout in milliseconds specifying time after which nRF Desktop DVFS module will retry DVFS frequency change.
```

Comment:
The timeout values are already described in section above (where you describe the handled error codes). No need to describe them again here.